### DISCUSSION TITLE
Squashes and updating my server

### DISCUSSION THREAD

**Discussion by Lumirion** [azerothcore/azerothcore-wotlk#14105] [General]
I haven't updated my server since last fall, maybe. Do I need to worry about database squashes?

**Comment by virgo77** [comment]
You say last fall... 2022 or 2021?

If 2022, what you have to do is a backup of your DB with HeidiSQL or SQLyog Community. Then, you need to update the "core" folder, then compile it again. After that, do a `make install` and launch authserver and check if some sql updates are applied. Quit authserver, and do the same with worldserver. If no errors occured, so you can play. And don't forget to check differencies between the .CONF.DIST files and your .CONF files. Some changes has been added.

If 2021, I think you will need to reinstall the DB. But export your DB first. Backup each one separetly (auth / characters / world / etc.). Then do a fresh install and reimport AUTH and CHARACTERS DB. Configure your CONF files to enable auto updates. Then, launch authserver, then worldserver. The sql updates must apply correctly. Please, don't forget to check [this link](https://github.com/wowgaming/client-data/releases/) as the maps/mmaps/vmaps was updated since last year.

NOTES:
- if you used mods, don't forget to reimport their sql files, and configure again the CONF files.
- what about me: I use WinSCP to sync files between my server (Debian 11) and my computer. I do a local "clone" before updating the core. And I backup my DB too. So I have a fresh backup if needed, and I can restore my server quickly.

Play well!

V.

--------------
AzerothCore Install Wiki: https://www.azerothcore.org/wiki/home
Check this one to be sure you are able to install and use the core depending on the versions of your OS, apps (make, git, boost, etc.) and also DB Manager (MySQL / MariaDB / other)

**Reply by Lumirion** [reply]
Ok. Thank you. It was last updated in 2021. Gonna give it a try, hopefully this weekend.
